doc/source/configuration/magnum-capi.rst

```diff
@@ -0,0 +1,36 @@
+========================
+Magnum Cluster API Driver
+========================
+
+Prequisites for deploying the CAPI driver in magnum: 
+
+Management Cluster
+==================
+The CAPI driver relies on a management cluster to manage tenant kubernetes clusters. 
```

Comment:
Can you please note that this is its self a Kubernetes cluster installed using cluster API, running inside the OpenStack cloud.